package.json

```diff
@@ -14,6 +14,7 @@
     "ejs": "^2.3.3",
     "express": "^4.13.3",
     "json-api-client": "~0.4.0",
+    "jsplumb": "1.7.9",
```

Comment:
Totally cool with just pulling down js-plumb after install as a vendor dependency. No need for this.